documentation/DevelopmentProcess/DevelopmentProcess.tex

```diff
@@ -84,28 +85,39 @@
 \section{Introduction}
 \subsection{Project Overview}
 SmartServe is an autonomous table tennis training system for a wide range of table tennis players to aid in diagnosing and improving their performance. The system does so by shooting balls toward the player and detecting successful returns. It can then adapt to the player's weaknesses and expose them by playing more to them over time. This alleviates problems of finding and working with a coach or training many players in a short amount of time. The system will be deemed a success if table tennis players enjoy and can see value in using our system in place of or in tandem to a coach.\\\\
-The development started at the beginning of the Fall 2017 academic turn and will conclude at the end of the Winter 2018 term. The team was assembled in the capstone course for Software and Mechatronics Engineering disciplines.
+The development started at the beginning of the Fall 2017 academic term and will conclude at the end of the Winter 2018 term. The team was assembled from MECHTRON 4TB6/SFWR ENG 4G06 Capstone course for the Software and Mechatronics Engineering disciplines.
 \subsection{Naming Conventions \& Terminology}
 \begin{itemize}
-\item \textbf{Git}: a distributed versioning control system
+\item \textbf{DevOps}: development operations
+\item \textbf{Git}: a version control system for tracking changes and coordinate work among multiple collaborators
 \item \textbf{Master Branch}: the main branch of the GitHub repository
 \item \textbf{PR}: pull request, a request to add changes from one branch into another done via a merge commit or rebase
 \item \textbf{Slack}: a messaging platform for teams to subscribe and publish to channels or send messages between two or more team members
-\item \textbf{DevOps}: development operations
+\item \textbf{Trello}: a web-based project management application utilized by the project team
 \end{itemize}
 
-\section{Team Members}
-The team members are as follows:
+\section{Project Team}
+The project team members consists of:
```

Comment:
The term "project team members" is in my opinion, hard to say. Could we change to "The team members for this project consists of:"?